packages/prosess-uttak/src/ui/Uttak.stories.tsx

```diff
@@ -211,251 +209,297 @@ export const Uttak: Story = {
+              arbeidstidprosent: 0,
+              løpendeInntekt: 0,
+              bruttoInntekt: 240000,
+              erNytt: false,
+            },
+          ],
+          beregningsgrunnlag: 720000,
+          løpendeInntekt: 600000,
+          bortfaltInntekt: 120000,
+          reduksjonsProsent: 80,
+          graderingsProsent: 20,
+        },
+      ],
+      arbeidsforhold: {
+        '111111111': {
+          identifikator: '111111111',
+          personIdentifikator: null,
+          navn: 'BEDRIFT 1 AS',
+          fødselsdato: null,
+          arbeidsforholdreferanser: [
+            {
+              internArbeidsforholdId: '724b3d31-c308-4ae9-9067-fd989b1881c4',
+              eksternArbeidsforholdId: 'ARB001',
+            },
+          ],
+        },
+        '222222222': {
+          identifikator: '222222222',
+          personIdentifikator: null,
+          navn: 'BEDRIFT 2 AS',
+          fødselsdato: null,
+          arbeidsforholdreferanser: [
+            {
+              internArbeidsforholdId: '4b426e33-aae0-4f04-b7a6-3fb70004b772',
+              eksternArbeidsforholdId: 'ARB002',
+            },
+          ],
+        },
+        '333333333': {
+          identifikator: '333333333',
+          personIdentifikator: null,
+          navn: 'BEDRIFT 3 AS',
+          fødselsdato: null,
+          arbeidsforholdreferanser: [
+            {
+              internArbeidsforholdId: '3f56315d-1f35-4657-b859-7c64b2ecdb23',
+              eksternArbeidsforholdId: 'ARB003',
+            },
+          ],
+        },
+      },
+      aksjonspunktkoder: [],
+      kodeverkUtenlandsoppholdÅrsak: [
+        {
+          kode: 'BARNET_INNLAGT_I_HELSEINSTITUSJON_DEKKET_ETTER_AVTALE_MED_ET_ANNET_LAND_OM_TRYGD',
+          kodeverk: 'UTENLANDSOPPHOLD_ÅRSAK',
+          navn: 'Barnet/den pleietrengende er innlagt i helseinstitusjon dekket etter avtale med annet land om trygd (mottar ytelse som i Norge, telles ikke i 8 uker)',
+        },
+        {
+          kode: 'BARNET_INNLAGT_I_HELSEINSTITUSJON_FOR_NORSK_OFFENTLIG_REGNING',
+          kodeverk: 'UTENLANDSOPPHOLD_ÅRSAK',
+          navn: 'Barnet/den pleietrengende er innlagt i helseinstitusjon for norsk offentlig regning (mottar ytelse som i Norge, telles ikke i 8 uker)',
+        },
+        {
+          kode: 'INGEN',
+          kodeverk: 'UTENLANDSOPPHOLD_ÅRSAK',
+          navn: 'Ingen årsak til utenlandsoppholdet er oppgitt, perioden telles i 8 uker',
+        },
+      ],
+      utsattePerioder: [],
+      erFagytelsetypeLivetsSluttfase: false,
+      løsAksjonspunktVurderDatoNyRegelUttak: undefined,
+      virkningsdatoUttakNyeRegler: null,
+      readOnly: false,
+    },
+  },
+  render: props => (
+    <FeatureTogglesContext.Provider value={{ BRUK_INNTEKTSGRADERING_I_UTTAK: true }}>
+      <UttakContainer {...props} />
+    </FeatureTogglesContext.Provider>
+  ),
+  play: async ({ canvasElement, step }) => {
+    const canvas = within(canvasElement);
```

Comment:
Du kan hente ut `canvas` direkte i linja over i stedet for `canvasElement`, så slipper du å gjøre `within` her